neuralprophet/conformal.py

```diff
@@ -167,29 +168,45 @@ def plot(self, plotting_backend: str):
                 specifies the plotting backend for the nonconformity scores plot, if any
 
                 Options
-                    * ``matplotlib``: Use matplotlib backend for plotting
-                    * ``plotly``: Use the plotly backend for plotting
+                * ``plotly-resampler``: Use the plotly backend for plotting in resample mode. This mode uses the
+                    plotly-resampler package to accelerate visualizing large data by resampling it. For some
+                    environments (colab, pycharm interpreter) plotly-resampler might not properly vizualise the figures.
+                    In this case, consider switching to 'plotly-auto'.
+                * ``plotly``: Use the plotly backend for plotting
+                * ``matplotlib``: use matplotlib for plotting
+                * (default) None: Plotting backend ist set automatically. Use plotly with resampling for jupyterlab
+                    notebooks and vscode notebooks. Automatically switch to plotly without resampling for all other
+                    environments.
 
         """
         method = self.method.upper() if "cqr" in self.method.lower() else self.method.title()
-        if plotting_backend == "plotly":
+        # Check whether a local or global plotting backend is set.
+        if hasattr(self, "plotting_backend") and plotting_backend is None:
+            plotting_backend = select_plotting_backend(self.plotting_backend)
+        else:
+            plotting_backend = select_plotting_backend(plotting_backend)
```

Comment:
What would happen if both conditions are false? Meaning no `self.plotting_backend` and no `plotting_backend` parameter? Then no `select_plotting_backend` is called 🤔

Reply:
@noxan Exactly, then `select_plotting_backend(None) `would be called. Which selects a plotting_backend based on the input which is None

Reply:
True, messed up some boolean logic in my head 👍